pkg/graphviz/traceflow.go

```diff
@@ -475,6 +484,64 @@ func GenGraph(tf *crdv1alpha1.Traceflow) (string, error) {
 		}
 		edge.Attrs[gographviz.Constraint] = "false"
 	}
-
+	if tf.Spec.LiveTraffic && tf.Status.Phase == crdv1alpha1.Succeeded {
+		err = createCapturedPacketNode(graph, cluster2.Name, tf)
+		if err != nil {
+			return "", err
+		}
+	}
 	return genOutput(graph, false), nil
 }
+
+func getCapturedPacketLabel(tf *crdv1alpha1.Traceflow) string {
+	label := "capturedPacket:\\ldstIP:" + tf.Status.CapturedPacket.DstIP + "\\l"
```

Comment:
Might look better to capitalize the words and use the full names, like: "Captured Packet", "Destination IP", "IPv6 Header", "Sequence", ...

Reply:
I just updated those words, please help to check again, thanks.